index.html

```diff
@@ -11,22 +11,23 @@
   <header>
     <h1>INTENTION TIMER</h1>
   </header>
+  <main>
```

Comment:
This is a good addition for differentiating the header from the rest of the page.